packages/shared/tsconfig.json

```diff
@@ -13,7 +13,11 @@
     "strictBindCallApply": true,
     "strictPropertyInitialization": true,
     "rootDir": "src",
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "baseUrl": ".",
+    "paths": {
+      "enterprise/license": ["packages/shared/src/enterprise/license.ts"]
+    }
```

Comment:
Not 100% sure about the implications of these tsconfig changes.  We should definitely test this in the production build as well as yarn dev since they use different typescript compilers.

Reply:
ah whoops, I had meant to remove that since it didn't work and I renamed the file as well.